Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement
 
 # Build stage: Install yarn dependencies
 # ===
-FROM node:19 AS yarn-dependencies
+FROM node:18 AS yarn-dependencies
```

Comment:
Does it not work with 19?

Reply:
I learned from vanilla, that we tend to use only LTS versions, hence I used 18 here.